packages/components/src/components/atoms/BaseCard/BaseCard.tsx

```diff
@@ -26,155 +44,79 @@ const useStyles = createUseStyles({
-        cardPosition.y === previousCardPosition.y
-      ) {
-        turnCard();
-      }
-    }, [cardPosition.x, cardPosition.y, disableNativeEvents, previousCardPosition.x, previousCardPosition.y, turnCard]);
-
-    const onKeyDown = (event: KeyboardEvent<HTMLDivElement>) =>
-      !disableNativeEvents && event.keyCode === 32 && turnCard();
-
-    const handleMouseDown = useCallback(
-      ({ clientX, clientY }) => {
-        if (!disableNativeEvents) {
-          setStartPosition({ x: clientX, y: clientY });
-          setPreviousCardPosition(cardPosition);
-          setIsDraging(true);
-        }
-      },
-      [disableNativeEvents, cardPosition]
-    );
-
-    const handleMouseMove = useCallback(
-      ({ clientX, clientY }) => {
-        const newCardPosition: Position = {
-          x: Math.round(cardPosition.x + clientX - startPosition.x),
-          y: Math.round(cardPosition.y + clientY - startPosition.y),
-        };
-        setPreviousCardPosition(cardPosition);
-        setCardPosition(newCardPosition);
-      },
-      // ⚠️ TODO: figure out why by putting cardPosition in dependencies it behaves weirdly
-      // eslint-disable-next-line react-hooks/exhaustive-deps
-      [startPosition]
-    );
-
-    const handleMouseUp = useCallback(() => {
-      setIsDraging(false);
-    }, []);
-
-    useEffect(() => {
-      if (isDragging) {
-        window.addEventListener('mousemove', handleMouseMove);
-        window.addEventListener('mouseup', handleMouseUp);
-      } else {
-        window.removeEventListener('mousemove', handleMouseMove);
-        window.removeEventListener('mouseup', handleMouseUp);
-
-        onPositionChanged(cardPosition);
-      }
-    }, [isDragging, handleMouseMove, cardPosition, onPositionChanged, handleMouseUp]);
-
-    return (
-      <div
-        data-testid="BaseCard"
-        role="button"
-        onClick={onClick}
-        onKeyDown={onKeyDown}
-        onMouseDown={handleMouseDown}
-        className={classnames(classes.card, { [classes.cardGrabbing]: isDragging, [classes.cardGrab]: !isDragging })}
-        tabIndex={0}
-      >
-        <div
-          data-testid="BaseCard-flipper"
-          className={classnames(classes.cardFlipper, { [classes.flipped]: !isFrontVisible })}
-        >
-          <div data-testid="BaseCard-front" className={classnames(classes.cardFront)}>
-            {frontFace}
-          </div>
-          <div data-testid="BaseCard-back" className={classnames(classes.cardBack)}>
-            {backFace}
-          </div>
+  return (
+    <Draggable
+      className={classNames(classes.card, className)}
+      position={position}
+      height={height}
+      width={width}
+      boundaries={boundaries}
+      onDragged={onPositionChanged}
+      onClick={() => onFlipped(!faceUp)}
+      disabled={disableNativeEvents}
+      // eslint-disable-next-line react/destructuring-assignment
+      data-testid={props['data-testid'] || 'BaseCard'}
```

Comment:
Shouldn't we create a getter for classname then?
I'd rather have a selector by classname than (perhaps even by class pattern, since jss generates random patterns)

Reply:
I disagree. Classes are an implementation detail and subjects to change, and thus I don't think it is a good way to identify DOM elements. To use `data-testid` was weird for me at the beginning but I think it is a good way to retrieve exactly the element you want to. Plus, we can remove it from production code if we want: https://www.npmjs.com/package/babel-plugin-jsx-remove-data-test-id